src/openloco/graphics/gfx.h

```diff
@@ -23,6 +25,23 @@ namespace openloco::gfx
 
     drawpixelinfo_t& screen_dpi();
 
+    struct loco_g1_header
+    {
+        uint32_t num_entries;
+        uint32_t total_size;
+    };
+
+     struct loco_g1_element
```

Comment:
`g1_element_t` seems to be more consistent with the rest of the codebase

Reply:
Can change it - I just followed OpenRCT2's `rct_g1_element`.

Reply:
So far I have used `_t` for any time that has no default initialisation. E.g. it won't be zeroed if you define a local instance of one.